app/Http/Resources/ProjectResource.php

```
@@ -16,11 +16,10 @@ public function toArray(Request $request): array
     {
         return [
             'id' => $this->id,
-            'code' => $this->code,
+            'project_code' => $this->project_code,
             'project_members' => $this->whenLoaded('project_has_employees', function ($value) {
                 return EmployeeUserResource::collection($value);
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Http\Resources\EmployeeUserResource' in method 'toArray'. (Clean Code Rules)

(StaticAccess)